RELEASENOTES.md

```diff
@@ -41,6 +41,77 @@ repository but is still accessible under the
 
 ---
 
+### Networking Recipe
+
+The networking recipe enhances and extends the functionality enabled in 23.01
+release.
+
+#### Features and enhancements
+
+* Intel IPU E2100 target support
+* IPsec offload support (Intel IPU E2100 target only)
+* IPv6 support in infrap4d networking stack (Intel IPU E2100 target only)
+* Local mirroring (Intel IPU E2100 target only)
```

Comment:
Please check with PM if we are aligning with 9.1 or 9.2. If it's 9.1, mirroring won't be available?

Reply:
Checked with @n-sandeep and he confirmed that the feature was available in 0.9.1. In 0.9.2, we switched from MatchValueLookupTable to MatchActionTable, but the feature exists in 0.9.1

Reply:
Sounds good, thanks Sabeel for checking